CryptoLightNews/Networking/Service.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 class Service {
     
     typealias News = Result<SearchResults, CryptoNewsErrors>
```

Comment:
News is not telling the truth about this type: you can have an error here. And SearchResults is not a good name for a group of Articles 😄 
```suggestion
    typealias ResultNews = Result<Articles, CryptoNewsErrors>
```